src/language/loader.py

```diff
@@ -0,0 +1,14 @@
+import importlib
+
+_cache = {}
+
+
+def load(language):
+    if language not in _cache.items():
```

Comment:
I think, this should be _cache.keys() here. Since .items() returns tuples, this line will always evaluate to True.

Like this:
```
d = {"one":1, "two":2, "three":3}
"one" not in d.items() 
```

will return True, even though "one" is in d.

Reply:
Wow, what a sneaky bug. Fixed.